### ISSUE TITLE
Error when updating PyMISP in AIL

### ISSUE THREAD

**Issue by robertnixon2003** [MISP/PyMISP#519]
I am getting this syntax error:

```
Collecting git+https://github.com/MISP/PyMISP
  Cloning https://github.com/MISP/PyMISP to /tmp/pip-req-build-jkk6r9yq
  Running command git clone -q https://github.com/MISP/PyMISP /tmp/pip-req-build-jkk6r9yq
  Running command git submodule update --init --recursive -q
    ERROR: Command errored out with exit status 1:
     command: /opt/AIL-framework//AILENV//bin/python3 -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-req-build-jkk6r9yq/setup.py'"'"'; __file__='"'"'/tmp/pip-req-build-jkk6r9yq/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' egg_info --egg-base /tmp/pip-req-build-jkk6r9yq/pip-egg-info
         cwd: /tmp/pip-req-build-jkk6r9yq/
    Complete output (10 lines):
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-req-build-jkk6r9yq/setup.py", line 7, in <module>
        import pymisp
      File "/tmp/pip-req-build-jkk6r9yq/pymisp/__init__.py", line 28, in <module>
        from .abstract import AbstractMISP, MISPEncode, pymisp_json_default, MISPTag, Distribution, ThreatLevel, Analysis  # noqa
      File "/tmp/pip-req-build-jkk6r9yq/pymisp/abstract.py", line 111
        self.__edited: bool = True  # As we create a new object, we assume it is edited
                     ^
    SyntaxError: invalid syntax
    ----------------------------------------
ERROR: Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.
```

**Comment by Rafiot** [comment]
You're using a python version below 3.6, it isn't supported by PyMISP any more. I'm surprised AIL still works.

**Comment by Rafiot** [comment]
Bump @Terrtia

**Comment by Terrtia** [comment]
Fixed in [AIL v3.0](https://github.com/CIRCL/AIL-framework/releases/tag/v3.0)

You can close it

**Comment by Rafiot** [comment]
Thank you!
